cwms-data-api/src/test/java/cwms/cda/api/ClobControllerTestIT.java

```diff
@@ -69,26 +64,41 @@ void test_create_getOne() throws JsonProcessingException, UnsupportedEncodingExc
             .assertThat()
             .statusCode(is(HttpServletResponse.SC_CREATED));
 
-        /* There is an issue with how javalin handles / in the path that are actually part
-        of the object name (NOTE: good candidate for actually having a GUID or other "code"
-        as part of the path and the actual name as a query parameter.
-        )
+//        /* There is an issue with how javalin handles / in the path that are actually part
```

Comment:
Odd additional line comments vs block comments.

Reply:
yeah, that is odd.